qunetsim/utils/constants.py

```diff
@@ -29,24 +29,24 @@ class Constants:
     NACK = 'qunetsim_NACK__'
 
     # PROTOCOL IDs
-    REC_EPR = 'rec_epr'
-    SEND_EPR = 'send_epr'
-    REC_TELEPORT = 'rec_teleport'
-    SEND_TELEPORT = 'send_teleport'
-    REC_SUPERDENSE = 'rec_superdense'
-    SEND_SUPERDENSE = 'send_superdense'
-    REC_CLASSICAL = 'rec_classical'
-    SEND_CLASSICAL = 'send_classical'
-    SEND_BROADCAST = 'send_broadcast'
-    RELAY = 'relay'
-    SEND_QUBIT = 'send_qubit'
-    REC_QUBIT = 'rec_qubit'
-    SEND_KEY = 'send_key'
-    REC_KEY = 'rec_key'
-    SEND_GHZ = 'send_ghz'
-    REC_GHZ = 'rec_ghz'
-    SEND_W = 'send_w'
-    REC_W = 'rec_w'
+    REC_EPR = 1
```

Comment:
Maybe we make these an enum or something...

Reply:
I first used enums, but I find the a bit cumbersome in pyhton, since you always have to use .value onto the enum element to get the "real" enum value.

Reply:
Makes sense, I think this is fine.